dockertest/subtestbase.py

```diff
@@ -155,8 +157,15 @@ def failif_not_in(needle, haystack, description=None):
             description = 'string'
         if needle in haystack:
             return
-        raise DockerTestFail("Expected string '%s' not in %s '%s'"
-                             % (needle, description, haystack))
+        expect_s = "Expected string '%s'" % needle
+        if '|' in needle:
+            needles = [n.strip() for n in needle.split('|')]
+            expect_s = "Expected strings %s" % needles
+            for subneedle in needles:
+                if subneedle.strip() in haystack:
+                    return
+        raise DockerTestFail("%s not in %s '%s'" %
+                             (expect_s, description, haystack))
```

Comment:
How about just adding a ``delimiter="|"`` parameter then?  That seems like a very small complexity increase that at least gives an interface to workaround related issues.  Though in general I agree with your statement, there isn't much justification for lots of extra complexity.  So I'm okay if you just want to merge this and be done with it.